CHANGELOG.md

```diff
@@ -114,7 +116,7 @@
 - [dev] Turns on "keychain sharing" to fix a keychain bug in sim - orta
 - GeneVC now shows about information, and trending artists - orta
 - GeneVC now shows artworks for the gene - orta
-- Show ‘bid now’ call to action on artworks that are in an auction - alloy
+- Show "bid now" call to action on artworks that are in an auction - alloy
```

Comment:
mainly so the spell checker didn't trigger on the - isn't worth fixing the spell checker yet